From 4bf9bde5f2c31e0b4d515489fe887011f4e03627 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH 74/90] recovery: remove asserts from updater-script for now

recovery: dont assert recovery for now

Flashing on all Nougat based recoverys is broken, and only devs flash cr6 for now

Change-Id: Ia25fc884933f5b9610837f028947a7f763163345
---
 tools/releasetools/ota_from_target_files.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/releasetools/ota_from_target_files.py b/tools/releasetools/ota_from_target_files.py
index 1c0015854..33e1ce6d1 100755
--- a/tools/releasetools/ota_from_target_files.py
+++ b/tools/releasetools/ota_from_target_files.py
@@ -773,15 +773,15 @@ def WriteFullOTAPackage(input_zip, output_file):
       metadata=metadata,
       info_dict=OPTIONS.info_dict)
 
-  assert HasRecoveryPatch(input_zip)
+  #assert HasRecoveryPatch(input_zip)
 
   # Assertions (e.g. downgrade check, device properties check).
   #ts = target_info.GetBuildProp("ro.build.date.utc")
   #ts_text = target_info.GetBuildProp("ro.build.date")
   #script.AssertOlderBuild(ts, ts_text)
 
-  target_info.WriteDeviceAssertions(script, OPTIONS.oem_no_mount)
-  device_specific.FullOTA_Assertions()
+  #target_info.WriteDeviceAssertions(script, OPTIONS.oem_no_mount)
+  #device_specific.FullOTA_Assertions()
 
   # Two-step package strategy (in chronological order, which is *not*
   # the order in which the generated script has things):
-- 
2.17.0

